### PR TITLE
Added incidence

### PR DESCRIPTION
**Pull request by sebasguts**

### CODE REVIEW

File: NormalizModule.cpp [hunk 1]
```diff
@@ -382,6 +382,19 @@ PyObject* NmzToPyNumber(renf_elem_class in)
...
 #endif
 
+PyObject* NmzToPyNumber(const dynamic_bitset& in)
+{
+    PyObject* result = PyList_New(0);
+    for (int i = 0; i < in.size(); i++) {
+        if (in[i]) {
+            PyObject* number = NmzToPyNumber(i);
+            PyList_Append(result, number);
+            Py_DecRef(number);
```
Comment (fingolfin) [inline NormalizModule.cpp]: @sebasguts Is there a reason you are not using `NmzBitsetToPyList`, which returns a list of bools, but instead write this new function which returns a list of integers?

**Comment by fingolfin** [comment]
@w-bruns you merged this, but I wonder if this is implements the behavior we want? See my comment in the code above. Thing is, once people start using this, we are stuck with it, unless we are willing to break people's code.

**Comment by w-bruns** [comment]
I will test it.

**Comment by fingolfin** [comment]
Note that the main concerns is about what the output should be: a list of booleans such as `[true,false,true,false]`, or a list of integers such `[1,3]`. Both have advantages and disadvantages, of course. But other bitsets are translated to lists of bools, so it's a bit odd that `Incidence` returns a list of integers instead. Mind you, I am not saying it is unreasonable, but I think that it should at least be justified, and documented.

**Comment by w-bruns** [comment]
I would prefer that incidence matrices are produced as 0-1-matrices. 0 and 1 need 1 character, whereas true needs 4 and false even 5.

**Comment by fingolfin** [comment]
Fine by me, but that's a third format now :-)

**Comment by w-bruns** [comment]
Let us see what the Sage people say.

**Pull request merged**